instrumentation/pulsar/pulsar-2.8/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/pulsar/v2_8/telemetry/MessageListenerContext.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.pulsar.v2_8.telemetry;
+
+/**
+ * Helper class for suppressing spans from the message "receive" operation when we know that the
```

Comment:
Not very clear to me. What I understand:

```suggestion
 * Context to kown if a message is being processed by a listener.
```

Reply:
I have reworked the comment and renamed the methods, could you see if it is better now.